src/coreEnforcer.ts

```diff
@@ -69,10 +79,19 @@ export class CoreEnforcer {
    */
   public loadModel(): void {
     this.model = newModel();
+    // this.model.synced = this instanceof SyncedEnforcer;
+    this.model.synced = false;
     this.model.loadModel(this.modelPath);
     this.model.printModel();
   }
 
+  /**
+   * get a new RoleManager based on the type of current Enforcer
+   */
+  public newRoleManager(): RoleManager {
```

Comment:
I don't think it's necessary, you should add setNamedRoleManager to this, like `setNamedRoleManager(group:string, roleManager: RoleManager)`.

Reply:
I didn't quite understand what you meaning. Do you mean
```
setNamedRoleManager(sync, rm)
```